src/Iot.Ingestion.RouterService/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="2.0.4" targetFramework="net452" />

```

Comment:
This package is [formally deprecated](https://www.nuget.org/packages/Microsoft.Azure.KeyVault.Core) and shouldn't be used in a sample.   We should move to the  `Azure.Security.KeyVault` family of packages.